.github/workflows/generated-macos-arm64-binary-wheel-nightly.yml

```diff
@@ -126,7 +126,7 @@ jobs:
       #       favor of GPU_ARCH_VERSION
       DESIRED_CUDA: cpu
       GPU_ARCH_TYPE: cpu
-      DOCKER_IMAGE: pytorch/manylinux-builder:cpu-main
+      DOCKER_IMAGE: pytorch/manylinux2_28-builder:cpu-main
```

Comment:
Please note, this is not used on MacOS arm64. TODO: Post a followup to remove this parameter